tests/test_io.py

```diff
@@ -149,3 +155,24 @@ def test_read_color_off():
 
     assert_points_xyz(color_off)
     assert_points_color(color_off)
+
+
+def test_read_bin():
+    arr = PyntCloud.from_file(data_path + '.bin')
```

Comment:
I can't find the ".bin" example file anywhere?? Did you remember to push it??

Reply:
I will make sure it is there when I push again! Perhaps it got caught by a .gitignore or something...